libraries/AP_DDS/AP_DDS_Service_Table.h

```diff
@@ -14,8 +14,8 @@ constexpr struct AP_DDS_Client::Service_table AP_DDS_Client::services[] = {
     {
         .req_id = to_underlying(ServiceIndex::ARMING_MOTORS),
         .rep_id = to_underlying(ServiceIndex::ARMING_MOTORS),
-        .srv_profile_label = "ArmMotorsService",
+        .srv_profile_label = "",
         .req_profile_label = "",
-        .rep_profile_label = "ArmMotors_Replier",
+        .rep_profile_label = "arm_motors__replier",
```

Comment:
Lets rename this to `arm_motors__rr` or something similar.Quite similar to how we do it the topic and data reader/writer profiler names

Reply:
@arshPratap - I'll make these changes in a follow up PR. Also would like to standardise the node name to lowercase and rename to `/ardupilot_dds`. There are a few other house-keeping changes as well, so may sweep them up in a PR other stylistic / non-functional edits.